packages/responsive/src/useMediaQueries.tsx

```diff
@@ -31,6 +31,8 @@ export const useMediaQueries = (
   // The `addEventListener` calls blow up legacy Edge (<= v18/pre chromium),
   // so we disable the functionality of updating after page load.
   const isLegacyEdge = navigator.userAgent.match(/Edge/)
+  const isUnsupportedSafari =
+    window.matchMedia("").addEventListener === undefined
```

Comment:
This seems like it could be a replacement for the `isLegacyEdge` conditional?

Reply:
@ActuallyACat I merged already but did we want to revert and do this ^

Reply:
No need to revert, if anything we can do a follow up PR for it, but it's no big deal